### DISCUSSION TITLE
Specify emulator to load a rom

### DISCUSSION THREAD

**Discussion by flagone** [OnionUI/Onion#530] [Ideas]
Is it possible to specify a specific emulator to load a rom?

My use case is that most of my Arcade roms will load with Mame 2003.  However a few need FB Neo.  I can load the FB Neo roms by placing them in the appropriate folder, then running the emulator under the Expert tab.

However, I would prefer to have all the Arcade roms together and just specify an override for the few roms that need FB Neo so that when I load them, FB Neo is used rather than Mame 2003.

**Comment by Olywa** [comment]
Unfortunately this is not currently possible due to the way the Stock OS (and hence onion) is coded & configured.  
I would be interested to know which games are working for you in FBNeo vs Mame ?

**Reply by flagone** [reply]
An example of a game is Demon Front.

**Reply by Olywa** [reply]
ah OK, yeah I don't think all PGM games are compatible with Mame 2003+. As I see it you have two options.

-Change your main 'Arcade' system core to use final burn and run everything there*
-Or add another 'Arcade' in your main systems/consoles menu (rather than expert), named something else (you can label one Mame and one FBN, give them diff icons and point them to different rom folders).

If either of these appeal I can talk you through the steps.

*you may not be aware but FBNeo is typically for more powerful/modern hardware and so not really recommended for the mini. While some games will certainly run, you might find they perform significantly better with FBAlpha2012 which will also run most things that Mame 2003+ will. (I'm not sure the fba2012 core is included by default at the moment (it used to be) but it's easy to add.

**Reply by flagone** [reply]
I think I tried Final Burn Alpha but it didn't run Demon Front.  At the moment, I can work around it by placing any roms that don't work in the FBNeo folder and running FBNeo from the Expert Tab.  Not ideal, but workable for me.

**Reply by Prophit219** [reply]
I'm actually interested in the steps. I'm sorta new to this and want to use the miyoo mini as an portable arcade machine. I would like to create arcade folders for specfic rom sets and run the corresponding core without going into expert mode. IE create a fba  arcade folder on the main menu and have fba2012 launch em. I can run some stuff off of retrocarch but I can't determine what the game name is from the zip file name.

**Comment by schmurtzm** [comment]
Now possible with [GLO Menu](https://onionui.github.io/docs/apps/game-list-options).
